tutorial/tests/test_input_output.py

```diff
@@ -2,17 +2,34 @@
 import csv
 import itertools
 import pathlib as pl
-import sys
-from io import StringIO
+from collections import Counter
+from io import StringIO, TextIOWrapper
 
 import pytest
 
 from .. import prepare_magic_file
 
 
 def get_data(name: str, data_dir: str = "data") -> pl.Path:
-    current_module = sys.modules[__name__]
-    return (pl.Path(current_module.__file__).parent / f"{data_dir}/{name}").resolve()
+    return (pl.Path.cwd() / f"tutorial/tests/{data_dir}/{name}").resolve()
```

Comment:
I'm not sure about this: the CWD might not correspond to ther location of the module...Otherwise I like this, it's very elegant

Reply:
`Path.cwd()` is the directory from where pytest is invoked, that is, `/some/base/path/python-tutorial`. It should work then. I'll double check before merging.